python/README.md

```diff
@@ -26,17 +30,22 @@ where `<your-platform>` is picked from `windows-x86_64`,`linux-x86_64`,`linux-x8
 `macosx-x86_64`, `linux-armhf` and `windows-x86_64-gpu`, depending on your operating system
 and architecture.
 
+`build_jar.py` uses **Python 3.4+**, but `konduit` itself is Python 2.7 compatible.
+
 ## Running tests
 
+Install test dependencies using `pip install 'konduit[tests]'` if you want to run tests. 
+
+On Windows, compiling the test dependencies requires Visual Studio Build Tools 14.0, which can be installed from [here](https://visualstudio.microsoft.com/downloads/). You may also need to install the Windows 8.1 / 10 SDK. See Python's [*WindowsCompilers*](https://wiki.python.org/moin/WindowsCompilers) page for details. 
```

Comment:
@philip-khor could you also please add a sentence describing that you need to download https://deeplearning4jblob.blob.core.windows.net/testresources/bert_mrpc_frozen_v1.zip, unzip it and place it the tests folder (preferably with curl, unzip and cp etc. to make it simple for the user)